content/en/storage-provider/skills/security/index.md

```diff
@@ -12,13 +12,21 @@ menu:
 weight: 630
 toc: true
 ---
-As a Filecoin Storage Provider you are not only storing customer data, you are also storing Filecoin wallets and running systems that require 24/7 uptime, so as not to lose collateral. In the event of a security intrusion on your network and systems being compromised, you risk downtime or even losing access to your systems or storage. Maintaining proper security is essential.
+Being a Filecoin Storage Provider involves more than just storing customer data. You are also responsible for managing Filecoin wallets and running systems that require 24/7 uptime to avoid losing collateral. This means that if your network or systems are compromised due to a security intrusion, you risk experiencing downtime or even losing access to your systems and storage. Therefore, maintaining proper security is of utmost importance.
```

Comment:
```suggestion

Being a Filecoin Storage Provider involves more than just storing customer data. You are also responsible for managing Filecoin wallets and running systems that require 24/7 uptime to avoid losing collateral. This means that if your network or systems are compromised due to a security intrusion, you risk experiencing downtime or even losing access to your systems and storage. Therefore, maintaining proper security is of utmost importance.
```

Keep an empty line after front-matter.